files/build_templates/per_namespace/lldp.timer

```diff
@@ -0,0 +1,12 @@
+[Unit]
+# This delay is for fast-reboot performance
+Description=Delays LLDP docker until SONiC has started
+PartOf=lldp.service
+
+[Timer]
+OnUnitActiveSec=0 sec
+OnBootSec=1min 30 sec
+Unit=lldp.service
```

Comment:
lldp.service is a per-namespace container. Will that work for multi-asic? Should we have several timers?

Reply:
Add  has_timer to this file https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/init_cfg.json.j2#L32

Reply:
Done